zpi-backend/src/main/java/pwr/zpibackend/services/thesis/ReservationService.java

```diff
@@ -131,6 +156,12 @@ private void removeExpiredReservations() {
                 .filter(reservation -> reservation.getReservationDate().isBefore(threshold))
                 .forEach(reservation -> {
                     reservationRepository.delete(reservation);
+
```

Comment:
Dodanie Async do metody

Reply:
jednak nie wolno, ludzie mówią, ze to i tak nie działa w ten sposób, tylko scheduled i tak juz dziala w pewien sposob na innym watku, wiec nie ma problemu z tym chyba